paper.bib

```diff
@@ -1,158 +1,154 @@
+    Doi = {10.3847/1538-3881/aabc4f},
+    Eid = {123},
+    Journal = {AJ},
+    Keywords = {methods: data analysis, methods: miscellaneous, methods: statistical, reference systems, Astrophysics - Instrumentation and Methods for Astrophysics},
+    Month = Sep,
+    Pages = {123},
+    Primaryclass = {astro-ph.IM},
+    Title = {{ The Astropy Project: Building an Open-science Project and Status of the v2.0 Core Package }},
+    Volume = {156},
+    Year = 2018,
+    Bdsk-Url-1 = {https://doi.org/10.3847/1538-3881/aabc4f}
 }
 
 @article{dalcin2005mpi,
-title={{MPI} for {Python}},
-author={Dalc{\'\i}n, Lisandro and Paz, Rodrigo and Storti, Mario},
-journal={Journal of Parallel and Distributed Computing},
-doi={10.1016/j.jpdc.2005.03.010},
-volume={65},
-number={9},
-pages={1108--1115},
-year={2005},
-publisher={Elsevier}
+    title = {{ MPI } for { Python }},
+    author = {Dalc { \'\i } n, Lisandro and Paz, Rodrigo and Storti, Mario},
+    journal = {Journal of Parallel and Distributed Computing},
+    doi = {10.1016/j.jpdc.2005.03.010},
+    volume = {65},
+    number = {9},
+    pages = {1108--1115},
+    year = {2005},
+    publisher = {Elsevier}
 }
 
 
 @article{hamaker_understanding_1996,
-	title = {Understanding radio polarimetry. {I}. {Mathematical} foundations},
-	volume = {117},
-	copyright = {© European Southern Observatory (ESO), 1996},
-	issn = {0365-0138, 1286-4846},
-	url = {https://doi.org/10.1051/aas:1996146},
-	doi = {10.1051/aas:1996146},
-	abstract = {Astronomy and Astrophysics, Supplement Series (A\&AS) published data papers, either observational or theoretical, as well as extensive data material forming the basis of papers with astrophysical results},
-	language = {en},
-	number = {1},
-	urldate = {2017-10-09},
-	journal = {Astronomy and Astrophysics Supplement Series},
-	author = {Hamaker, J. P. and Bregman, J. D. and Sault, R. J.},
-	month = may,
-	year = {1996},
-	pages = {137--147}
+    title = {Understanding radio polarimetry. { I } . { Mathematical } foundations},
+    volume = {117},
+    copyright = {© European Southern Observatory (ESO), 1996},
+    issn = {0365-0138, 1286-4846},
+    url = {https://doi.org/10.1051/aas:1996146},
+    doi = {10.1051/aas:1996146},
+    abstract = {Astronomy and Astrophysics, Supplement Series (A\&AS) published data papers, either observational or theoretical, as well as extensive data material forming the basis of papers with astrophysical results},
+    language = {en},
+    number = {1},
+    urldate = {2017-10-09},
+    journal = {Astronomy and Astrophysics Supplement Series},
+    author = {Hamaker, J. P. and Bregman, J. D. and Sault, R. J.},
+    month = may,
+    year = {1996},
+    pages = {137--147}
```

Comment:
I'm disinclined to change this file. It's for the accepted JOSS paper and I don't see the utility in changing it (it could also possibly mess things up?)

Reply:
Sure, I'll revert it.